content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -120,33 +118,44 @@ sure that its parent commit is pushed to Git.
 
 </admon>
 
-Updates to the live metrics are highlighted in orange in the
+Updates to the live metrics are highlighted (in orange) in the project table and
 [compare pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#compare-experiments)
-as well.
+in real time.
 
-The number of experiments with new updates to the live metrics values are
-displayed in the `Live` icon, which can also be used to filter and show only
-running experiments in the table.
+The number of experiments with recent updates to live metrics are displayed in
+the `Live` icon, which can also be used to filter and show only running
+experiments in the table.
 
 Live plots are displayed in the
-[plots pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#how-to-generate-plots);
-you can see the plots getting populated as Studio receives new updates.
+[plots pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#how-to-generate-plots).
+You can see them getting populated as Studio receives new updates.
 
 ![](https://static.iterative.ai/img/studio/live_plots.gif)
 
 An experiment can have one of the following statuses:
 
 - **Running** - Iterative Studio expects to receive live metrics and plots for
-  these experiments. Note that if the experiment stops due to any error,
-  Iterative Studio will not be aware of this and it will continue to wait for
-  live updates. In this case, you should delete the live metrics row from the
-  project table in Iterative Studio.
+  these experiments.
+
+  <admon type="warn">
+
+  Note that if the experiment stops due to any error, Iterative Studio will not
+  be aware of this and it will continue to wait for live updates. In this case,
+  you can delete the row from the project table.
+
+  </admon>
+
 - **Completed** - Iterative Studio does not expect to receive any more updates
-  for these experiments. Once the experiment concludes, you can delete the live
-  metrics row from the project table. Iterative Studio does not automatically
-  commit and push the final results of your experiment to Git. If you want to
-  save the experiment result, you should persist it using appropriate Git
-  commands.
+  for these experiments. Once the experiment concludes, you can delete the row
+  from the project table.
+
+  <admon type="warn">
+
+  Iterative Studio does not automatically commit and push the final results of
+  your experiment to Git. If you want to save the experiment result, you should
+  make it [persistent] it using appropriate DVC and Git commands.
```

Comment:
Can't make exps persistent from Studio?

Reply:
What about https://dvc.org/doc/studio/user-guide/projects-and-experiments/run-experiments#submit-a-new-experiment ?

Reply:
No, atm we can't make existing exps persistent from Studio. `submit-a-new-experiment` is about creating a new experiment altogether (and triggering the CI job if it is configured), not about taking an existing experiment run and persisting it without triggering a new CI/experiment job. Persisting is needed though, and it's been discussed - not sure if we've created an issue for this. cc mvshmakov

Reply:
is `you should make it [persistent] it using appropriate DVC and Git commands.` correct? Is the second it needed?

Reply:
Fixed.